src/main/java/com/mixpanel/android/viewcrawler/EditProtocol.java

```diff
@@ -52,9 +52,14 @@ public CantGetEditAssetsException(String message, Throwable cause) {
         }
     }
 
-    public EditProtocol(ResourceIds resourceIds, ImageStore imageStore) {
+    public interface OnErrorListener {
+        public void sendErrorMessage(ViewVisitor.CantVisitException e);
```

Comment:
By convention, this method should be named onError (to go with the classname, OnErrorListener)